### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by logonoff**
todo
- [ ] test favicon
- [ ] test titles

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
   env: {
     browser: true,
-    es2021: true
```
Comment (ggggg) [inline .eslintrc.js:4]: why

File: src/components/Team/Team.jsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { Typography } from '@mui/material';
 
 import DefaultUser from '../../assets/graphics/default_user.svg';
 import {
-  SkeletonLoadedImage,
```
Comment (ggggg) [inline src/components/Team/Team.jsx:7]: weird format but k

File: src/components/index.jsx [hunk 3]
```diff
@@ -17,5 +17,5 @@ export { NamedIconButton } from './NamedIconButton/NamedIconButton';
 export { Navbar } from './Navbar/Navbar';
 export { RouterBreadcrumb } from './RouterBreadcrumb/RouterBreadcrumb';
 export { SkeletonLoadedImage } from './SkeletonLoadedImage/SkeletonLoadedImage';
-export { TableOfContents } from './TableOfContents/TableOfContents';
+// export { TableOfContents } from './TableOfContents/TableOfContents';
```
Comment (ggggg) [inline src/components/index.jsx:20]: code comments

**Review by ggggg — CHANGES_REQUESTED** [review]

**Review by ggggg — APPROVED** [review]
Slightly skimmed it, lgtm

**Review by ggggg — APPROVED** [review]
Slightly skimmed it, lgtm

**Review by huonggiangbui — APPROVED** [review]
i trust u

**Pull request merged**